LayoutTests/platform/mac-wk1/TestExpectations

```diff
@@ -860,6 +860,8 @@ webkit.org/b/246719 imported/w3c/web-platform-tests/reporting/generateTestReport
 webkit.org/b/246719 imported/w3c/web-platform-tests/reporting/cross-origin-report-no-credentials.https.sub.html [ Failure ]
 webkit.org/b/246719 imported/w3c/web-platform-tests/reporting/cross-origin-reports-isolated.https.sub.html [ Failure ]
 
+webkit.org/b/285098 imported/w3c/web-platform-tests/content-security-policy/report-hash/ [ Failure ]
```

Comment:
Can we include a comment explaining why this is here?

Reply:
Done